tests/python/frontend/paddlepaddle/test_forward.py

```diff
@@ -2302,5 +2302,112 @@ def forward(self, x, y):
     verify_model(Dist(), input_data=[y, v])
 
 
+@tvm.testing.uses_gpu
+def test_forward_affine_channel():
+    class AffineChannel(nn.Layer):
+        def __init__(self):
+            super(AffineChannel, self).__init__()
+
+        @paddle.jit.to_static
+        def forward(self, inputs, scale, bias):
+            return paddle.fluid.layers.affine_channel(inputs, scale, bias)
+
```

Comment:
`fluid.layers.xxx` is deprecated, and will be removed in next version, please use other api, or just remove this operator